### PR TITLE
CAPA Maintainer promotions

### PR DESCRIPTION
**Pull request by randomvariable**
Promote @richardcase to Cluster API AWS wide maintainer status, removing the special EKS only designation.
Migrate @chuckha to emeritus status

Also additionally bring SIG and Cluster API leads into sync

Marking as WIP in case anyone wants to step up as reviewers whilst we're here. @michaelbeaumont , @mytunguyen  , @dthorsen  ?

/hold

### CODE REVIEW

**Comment by dthorsen** [comment]
Thanks for asking @randomvariable. I'd be interested in helping out with reviews. I think I still need sponsorship to join the kubernetes-sigs org though.

**Comment by richardcase** [comment]
> Thanks for asking @randomvariable. I'd be interested in helping out with reviews. I think I still need sponsorship to join the kubernetes-sigs org though.

I'd be happy to be one of the sponsors if you raise the request

**Comment by randomvariable** [comment]
@dthorsen Yup, please file at https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E and I'd be happy to sponsor.

**Comment by sedefsavas** [comment]
I'd be interested in helping with the reviews and added to reviewers list.
https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2116https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2116

**Comment by richardcase** [comment]
> I'd be interested in helping with the reviews and added to reviewers list.

+1 on adding @sedefsavas and @dthorsen as reviewers.

**Comment by randomvariable** [comment]
Agreed to merge on CAPA meeting of Monday 30 Nov .

/assign @vincepri 
/assign @detiber 
/assign @ncdc

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -10,7 +10,22 @@
...
+fabriziopandini
+## End SIG Cluster Lifecycle Leads
+
+## Start Cluster API Admins correct as of 2020/10/14 (deduped from above)
+davidewatson
+detiber
+vincepri
+## End Cluster API admins
+
+## Start Cluster API Provider AWS Admins (deduped from above)
```
Comment (sedefsavas) [inline SECURITY_CONTACTS:26]: I don't think we have admins in this repo?
```suggestion
## Start Cluster API Provider AWS Maintainers (deduped from above)
```

**Review by vincepri — APPROVED** [review]
/approve
/lgtm

**Pull request merged**